docs/gettingstarted/configuration.rst

```diff
@@ -76,3 +76,91 @@ Environment variables
 
 ``DEFAULT_COUNTRY``
   Sets the default country for the store. It controls the default VAT to be shown if required, the default shipping country, etc.
+
+``TIME_ZONE``
+  Sets the time zone. Defaults to `UTC`.
+
+``LANGUAGE_CODE``
+  Sets the language. Defaults to `en`.
+  See [more documentation](https://docs.djangoproject.com/en/2.1/topics/i18n/translation/#how-django-discovers-language-preference))
+
+``USE_I18N``
+  Whether or not to enable Djangos translation system I18N. Defaults to `True`.
+  See [more documentation](https://docs.djangoproject.com/en/2.1/ref/settings/#use-i18n)
+
+``USE_L10N``
+  Whether or not to use localization - eg date formatting. Defaults to `True`.
+  See [more documentation](https://docs.djangoproject.com/en/2.1/ref/settings/#use-l10n)
+
+``USE_TZ``
+  Enable or disable timezone awareness. Defaults to `True`
+  See [more documentation](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-USE_TZ)
+
+``ROOT_LOG_LEVEL``
+  The 'root' log level defaults to 'INFO'
+
+``MAIL_ADMINS_HANDLER_LEVEL``
+  The mail admin handler log level, defaults to `ERROR`
+
+``CONSOLE_HANDLER_LEVEL``
+  The console handler log level, defaults to `DEBUG`
+
+``DJANGO_LOG_LEVEL``
+  The django log level, defaults to `INFO`
+
+``DJANGO_SERVER_LOG_LEVEL``
+  The django.server log level, defaults to `INFO`
+
+``SALEOR_LOG_LEVEL``
+  The saleor log level, defaults to `DEBUG`
+
+``AVAILABLE_CURRENCIES``
+  A comma separated list of the currencies to be available for selection.
+
+``ACCOUNT_ACTIVATION_DAYS``
+
+``LOGIN_REDIRECT_URL``
+  Redirect url on unsuccessful login attempts
+
+``LOW_STOCK_THRESHOLD``
+  Low stock flag threshold for products
+
+``MAX_CART_LINE_QUANTITY``
+
+``PAGINATE_BY``
+  Number of products per page to display in public customer facing site
+
+``DASHBOARD_PAGINATE_BY``
+  Number of products per page to display in the admin dashboard
+
+``DASHBOARD_SEARCH_LIMIT``
+
+``PLACEHOLDER_IMAGE_60``
+  The placeholder image url of size 60 x 60
+
+``PLACEHOLDER_IMAGE_120``
+  The placeholder image url of size 120 x 120
+
+``PLACEHOLDER_IMAGE_255``
+  The placeholder image url of size 255 x 255
+
+``PLACEHOLDER_IMAGE_540``
+  The placeholder image url of size 540 x 540
+
+``PLACEHOLDER_IMAGE_1080``
+  The placeholder image url of size 1080 x 1080
+
+``DEFAULT_PLACEHOLDER``
+  The url for the default placeholder. Defaults to the `PLACEHOLDER_IMAGE_255` url.
+
+``LOGOUT_ON_PASSWORD_CHANGE``
+  Whether one should be logged out after changing one's password. Defaults to `False`
+
+``DB_SEARCH_ENABLED``
+  Whether or not to enable the search queries in the database. Will be superceded
+  by elasticsearch if elasticsearch is enabled. Defaults to `True`
+
+``SOCIAL_AUTH_USERNAME_IS_FULL_EMAIL``
+
+``NOCAPTCHA``
+  Whether to activate google's nocaptha feature for logging in.
```

Comment:
As the comment says in the `settings`, it's not a toggle for the Google's noCaptcha feature, at all. The value should not, by any mean be changed to `False`. The old Google's captcha (reCaptcha) is no longer existing and supported by Google.

To enable the noCaptcha feature ("Are you human? Just check the box the prove it."), you need to provide API keys, otherwise it stays disabled.

Reply:
Ah, you're right, I forgot that after the break !